tasks/task_manager.hh

```diff
@@ -100,6 +100,8 @@ public:
             abort_source _as;
         public:
             impl(module_ptr module, task_id id, uint64_t sequence_number, std::string keyspace, std::string table, std::string entity, task_id parent_id) noexcept;
+            impl(const impl&) = delete;
+            impl(impl&&) = delete;
```

Comment:
Please add a comment explaining why, documenting that task_manager::task::impl is always created as a unique_ptr hence it need not move or be copied.